ios/brave-ios/Sources/Brave/Helpers/URLBarHelperConstants.swift

```diff
@@ -0,0 +1,1149 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+struct URLBarHelperConstants {
+
```

Comment:
this whole thing needs a comment explaining what's going on

Reply:
Transferred any comments they have in search repo but the hash values are just in a separate file and no documentation.